### DISCUSSION TITLE
[Feature] Hidden Albums

### DISCUSSION THREAD

**Discussion by funkemunky** [immich-app/immich#13702] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The feature is quite simplistic in nature and should not be complicated to implement, and I will help plan and make contributions if it is accepted by maintainers.

The Feature: Add an option for albums to be "hidden".

Requirements of "hidden" albums:
* Photos in the album are removed from indexing of search, memories, and tagging.
* Photos are not viewable in main view.
* The only way to view the photos would be to navigate to the hidden album.

Additional functionality of this feature:
* Users can put a passcode on "Hidden" albums.
* Separate menu under albums to view "Hidden" albums.

Let me know what you think and any additional requirements that would make this feature more complete.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by Yoda8myhead** [comment]
I would love this feature. I use the secure folder on my phone and the locked folder on google images, and would like a similar feature on my immich server, which I'm trying to move to entirely.

**Comment by bo0tzz** [comment]
#2451
